javascript/javascript2/week1/spritname.js

```diff
@@ -0,0 +1,2 @@
+let name = window.prompt('Write your name');
+document.write(name + ', The fullmoon wolf.');
```

Comment:
The exercise requires you to add a **button** element and write a function that has to be called when you click on the button